plugin/github_test.go

```diff
@@ -0,0 +1,81 @@
+package plugin
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMain(m *testing.M) {
+	// Makes related environments empty,
+	// and reset these after test
+	origTokenEnv := os.Getenv("GITHUB_TOKEN")
+	defer os.Setenv("GITHUB_TOKEN", origTokenEnv)
```

Comment:
[nit]
Although It is polite and good, unfortunately this `defer` does not work because of `os.Exit`.
ref. https://golang.org/pkg/os/?m=all#Exit
It is confusing, so you should remove it.

(Be careful as the `defer` does not work while using` log.Fatal` or `panic` as well.)

Since the test runs in a child process, you may not need to restore the original value, or you may carefully `unset` environment variables for each test case.

Reply:
oops. I misunderstood...  I fixed it by https://github.com/mackerelio/mkr/pull/123/commits/73ffd1bada62514adae0830aa4b796fb169bd1e8